nautobot/core/testing/filters.py

```diff
@@ -119,6 +122,31 @@ def test_filters_generic(self):
                     )
                 This expects a field named `devices` on the model and a filter named `devices` on the filterset.
             """
+            if getattr(self.queryset.model, "is_contact_associable_model", False):
+                for generic_filter_test in (
+                    ["contacts", "associated_contacts__contact__name"],
+                    ["contacts", "associated_contacts__contact__id"],
+                    ["teams", "associated_contacts__team__name"],
+                    ["teams", "associated_contacts__team__id"],
+                ):
+                    if generic_filter_test not in self.generic_filter_tests:
+                        self.generic_filter_tests = (*self.generic_filter_tests, generic_filter_test)
+
+                # Make sure we have some valid contact-associations:
+                for contact, team, instance in zip(Contact.objects.all()[:3], Team.objects.all()[:3], self.queryset):
```

Comment:
This probably needs to create Contact and Team objects if none exist (as will likely be the case in app testing).